tests/System.Buffers.Primitives.Tests/CastTests.cs

```diff
@@ -56,7 +58,7 @@ public void IntArraySpanCastedToByteArraySpanHasSameBytesAsOriginalArray()
             var ints = new int[100000];
             Random r = new Random(42324232);
             for (int i = 0; i < ints.Length; i++) { ints[i] = r.Next(); }
-            var bytes = ints.AsSpan().NonPortableCast<int, byte>();
+            var bytes = MemoryMarshal.Cast<int, byte>(ints.AsSpan());
```

Comment:
Why are you changing it to Cast and not to TryCast?

Reply:
It wasn't TryCast before. And we don't have that yet.